tests/test_definitions/multicolumn_map_expectations/expect_compound_columns_to_be_unique.json

```diff
@@ -1,6 +1,84 @@
 {
   "expectation_type" : "expect_compound_columns_to_be_unique",
   "datasets" : [{
+    "data" : {
```

Comment:
@billdirks Assuming that the tests pass, this looks good.  Are we going to test mostly with the other two multi-column expectation `JSON` examples (seems like might be useful, if not too much work)?  Also, I am curious about how we are ensuring that the V2 tests do not break.  Thanks!

Reply:
@billdirks Does V2 accept `mostly` for multi-column expectations?  If yes, then your fix was probably a nice way to proceed.  However, if it does not, then we may have to do more investigation, because without processing `mostly`, the V2 tests using the same `JSON` test cases should not succeed (not all of them, at least).  Happy to discuss.  Thanks!

Reply:
V2 does accept the `mostly` argument for multi-column expectations! It doesn't do validation so my V2 validation tests failed which led me to add this.